enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/configuration/EnterpriseEditionSettings.java

```diff
@@ -32,7 +32,7 @@
 public class EnterpriseEditionSettings
 {
     @Description( "Specified names of id types (comma separated) that should be reused. " +
-                  "Currently only 'RELATIONSHIP' type is supported. " )
```

Comment:
Right, it will not enable node id reuse. It might not be entirely safe to enable node/relationship id reuse in 2.3. We should probably do it for 3.1.